Automattic/wp-calypso
# Update the link to the cancel FAQ

#### Changes proposed in this Pull Request

We need to update the anchor that the "cancel your domain" links to from the TOS section on the purchase page.

#### Testing instructions

Add a domain to the cart, on the purchase page, click on the "cancel your domain" link in the "By checking out:" section.
Make sure that the link takes you to: https://wordpress.com/support/manage-purchases/#cancel-a-purchase


https://user-images.githubusercontent.com/1379730/125491906-27fd97d0-2697-40cf-a17d-0c94c2e22782.mov
